src/videojs.ga.coffee

```diff
@@ -66,11 +68,13 @@ videojs.plugin 'ga', (options = {}) ->
         seeking = true
         sendbeacon( 'seek start', false, seekStart )
         sendbeacon( 'seek end', false, seekEnd )
+        percentsAlreadyTracked = [];
```

Comment:
Why do you reset the `percentsAlreadyTracked`?
Also, the semi colon is not required in coffee.